### PR TITLE
Add desc about flash options in flashSpec.xml

### PR DESCRIPTION
**Pull request by junmin-kim**
Add below description about flash options
ALL : Download all binaries and images
ERASE_USERFS : Erase user file system

### CODE REVIEW

**Comment by junmin-kim** [comment]
I can change if you have a better description

File: build/configs/artik053/.flashSpec.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
         <execute name="flash" path="${PROJECT_PATH}/os" command="make download" os="linux" bit="all"></execute>
     </executors>
     <options>
-        <option>ALL</option>
-        <option>ERASE_USERFS</option>
+        <option desc="Download all binaries and images">ALL</option>
+        <option desc="Erase user file system">ERASE_USERFS</option>
```
Comment (sunghan-chang) [inline build/configs/artik053/.flashSpec.xml]: partition is better than user file system

File: build/configs/artik053/.flashSpec.xml [hunk 2]
```diff
@@ -20,7 +20,7 @@
         <execute name="flash" path="${PROJECT_PATH}/os" command="make download" os="linux" bit="all"></execute>
     </executors>
     <options>
-        <option>ALL</option>
-        <option>ERASE_USERFS</option>
+        <option desc="Download all binaries and images">ALL</option>
```
Comment (sunghan-chang) [inline build/configs/artik053/.flashSpec.xml]: use flash instead of download

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**